src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -68,6 +69,9 @@ public static async Task OpenPropertiesWindowAsync(object item, IShellPage assoc
                     var propertiesWindow = new WinUIEx.WindowEx();
                     var appWindow = propertiesWindow.AppWindow;
 
+                    // Set icon
+                    appWindow.SetIcon(Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles/StoreLogo.ico"));
```

Comment:
We'd like to use a method that works with png but happy to fall back on this if there isn't any other choice.

We should however have an icon for each version of the app (dev, preview and release)